libs/utils/android/system.py

```diff
@@ -63,9 +76,9 @@ def systrace_start(env, trace_file, time=None,
                 return None
 
         #  Format the command according to the specified arguments
-        systrace_pattern = "{} -e {} -o {} {}"
+        systrace_pattern = "{} -e {} -o {} -b {} {}"
```

Comment:
Not sure systrace can really use the buffer parameter, at least when atrace is used on the target side... did you already verified that?